controllers/tagController.js

```diff
@@ -0,0 +1,38 @@
+const tags = require('../models/tags');
+
+// an onclick function will place the entry_id and the specific tag that was clicked on in the req body
+const add = (req,res) =>{
+    const { tag,entry_id } = req.body
+    tags.createTag(entry_id,tag)
```

Comment:
Our style guide doesn't accept 4 spaces for indentation. Be sure to lint your code for formatting AND most importantly for identifying certain code patterns :)